docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -115,6 +117,13 @@ Property Name               Default Value    Description
 Currently the connector only supports ``Log`` and ``MergeTree`` table engines
 in create table statement. ``ReplicatedMergeTree`` engine is not yet supported.
```

Comment:
``CREATE TABLE``

Reply:
Pre-existing. Noted down, will fix in a later change. I'd like to keep this PR isolated to just mechanical splitting of things without changing unrelated things.